### ISSUE TITLE
Custom Views in 2.0

### ISSUE THREAD

**Issue by leastprivilege** [IdentityServer/Documentation#103]
The current docs are hard to follow - I would recommend structuring it according to the amount of customization that can be done (at least that's how I explain it to people).

* Just changing styles, color, fonts etc (adding, replacing CSS)
* Using custom templates
* Using custom templates with DefaultViewService customization
* Writing a custom ViewService

**Comment by brockallen** [comment]
Ok done
